benthos-plugins/docs/antora.yml

```diff
@@ -0,0 +1,2 @@
+name: redpanda-labs
```

Comment:
should this be redpanda-connect-labs? 
Or we're putting it together with other RP stuff?

Reply:
It's a requirement for this to appear on the site here: https://docs.redpanda.com/redpanda-labs/

Reply:
Yeah I was just wondering if it would make sense to split connect labs into their own page.

Reply:
I think we should have a single labs site and repo.